spec/xclarity_client_storage_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'spec_helper'
+
+describe XClarityClient do
+  before :all do
+    WebMock.allow_net_connect! #-- Uncomment this line if you're testing with a external mock.
+  end
+
+  let(:conf) do
+    XClarityClient::Configuration.new(
+      username:   ENV['LXCA_USERNAME'],
+      password:   ENV['LXCA_PASSWORD'],
+      host:       ENV['LXCA_HOST'],
+      port:       ENV['LXCA_PORT'],
+      auth_type:  ENV['LXCA_AUTH_TYPE'],
+      verify_ssl: ENV['LXCA_VERIFY_SSL']
+    )
+  end
+
+  let(:client) { XClarityClient::Client.new conf }
+
+  it 'has a version number' do
+    expect(XClarityClient::VERSION).not_to be nil
+  end
+
+  context 'with discover_storages' do
+    before do
+      @storages = client.discover_storages
+    end
+
+    describe 'GET /storage' do
+      it 'should respond with an array' do
+        expect(@storages.class).to eq(Array)
+        expect(@storages.any?).to eq(true)
+      end
+
+      it 'correct response values' do
+        @storages.each do |storage|
```

Comment:
__Required:__
Here do you want to check just the data of one storage, so would be good if you access this one storage node with the index of the element in the array (in this case `[0]`), once that, if you add a second storage node in your request mock, this test will wrongly fail.

Reply:
Thanks for the sugestion, I fixed that.